### ISSUE TITLE
Organize import adds useless import

### ISSUE THREAD

**Issue by tivervac** [eclipse/xtext-xtend#536]
When you have code that has a static extension import, and you use that imported function in a static AND an extended way, an extra useless import is added.

```
package pkg;

import org.eclipse.emf.ecore.EObject

import static extension org.eclipse.emf.ecore.util.EcoreUtil.getURI

class Test {
	
	def void doSomething() {
		val EObject e = null
		val a = e.URI // Extended call
		val b = getURI(e) // Static call
	}
}
```
Organize import on the above code returns:
```
package pkg;

import org.eclipse.emf.ecore.EObject

import static org.eclipse.emf.ecore.util.EcoreUtil.* // This is useless

import static extension org.eclipse.emf.ecore.util.EcoreUtil.getURI

class Test {
	
	def void doSomething() {
		val EObject e = null
		val a = e.URI // Extended call
		val b = getURI(e) // Static call
	}
}
```

**Comment by ArneDeutsch** [comment]
Added a check if the static import is already covered by a static extension import. If so, do not add the static import for that type.

**Comment by cdietrich** [comment]
Fixed in 2.15. thx @ArneDeutsch
